nrf_rpc/doc/usage.rst

```diff
@@ -24,7 +24,7 @@ RPC encoders
 
 RPC encoders encode commands and events into serialized packets.
 Creating an encoder is similar for all packet types.
-The first step is the allocation of a buffer using :c:macro:`NRF_RPC_ALLOC`.
```

Comment:
Suggesting numbered steps. Maybe start like this: 
To create an encoder, complete the following steps:

1. Allocate a buffer using ...
#. Encode the parameters directly ...

The packet is sent using one of the ... (not anymore a step, because this is something that happens after 1 and 2)

Reply:
Done